packages/lesswrong/components/tagging/TagSubforumPage2.tsx

```diff
@@ -344,6 +345,12 @@ const TagSubforumPage2 = ({classes}: {
     skip: !tag
   })
 
+  const recordSubforumView = useRecordSubforumView({userId: currentUser?._id, tagId: tag?._id});
+  useEffect(() => {
+    if (!loadingTag && tag?._id)
+      void recordSubforumView();
+  }, [loadingTag, recordSubforumView, tag?._id]);
```

Comment:
This is required to fix a bug where the notifications menu wouldn't show up when you first joined because there was no UserTagRel object